### PR TITLE
RELATED: RAIL-4068 Remove parameter optionality wording from tsdoc

### PR DESCRIPTION
**Pull request by plavemar**
JIRA: RAIL-4068

---

Supported PR commands:

| Command                  | Description             |
| ------------------------ | ----------------------- |
| `ok to test`             | Re-run standard checks  |
| `extended test`          | BackstopJS tests        |
| `extended check sonar`   | SonarQube tests         |
| `extended check cypress` | Cypress E2E tests       |
| `extended check plugins` | Dashboard plugins tests |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [ ] `check` passes
-   [ ] `check-extended` passes
-   [ ] `check-extended-cypress` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/7441/ : SUCCESS in 11s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/9290/ : SUCCESS in 17m 25s

File: libs/api-client-bear/src/metadata.ts [hunk 1]
```diff
@@ -496,8 +496,11 @@ export class MetadataModule {
      * Returns all analytical dashboards (kpi dashboards) in a project specified by the given projectId
      *
      * @param projectId - Project identifier
-     * @param fetchAllListedDashboards - Optionally specify if also all the listed
-     *  dashboards should be loaded. Note that these include not just shared dashboards and dashboards
+     * @param fetchAllListedDashboards - Specify if also all the listed
+     *  dashboards should be loaded.
+     *
+     * @remarks
```
Comment (no23reason) [inline libs/api-client-bear/src/metadata.ts:502]: This is not correct, the whole text was for the parameter. If you want to move it to remarks, please reword it s that the note is linked to the parameter.

**Review by no23reason — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/2153/ : SUCCESS in 1m 12s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/2439/ : SUCCESS in 20s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-gate/1261/ : SUCCESS in 13m 56s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/9296/ : SUCCESS in 16m 58s

**Pull request merged**